CVE-2023-0386/ubuntu.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: Update linux kernel and install the vulnerable service
+  hosts: all
+  become: yes
+
+  tasks:
+    - name: Install dependencies
+      apt:
+        name:
+          - pkg-config
+          - libfuse-dev
+          - libcap-dev
+          - tmux
+        state: present
+        update_cache: yes
+
+    - name: Copy data to VM
+      copy:
+        src: ./data/
+        dest: /usr/local/bin/
+        mode: '0777'
+    
+    # Unluckily 5.15.91 and after won't work, 5.11-5.15.90 && 5.16-5.19 works tho.
+    # it comes with 5.15.125 (as of 11/18/2024)
+    - name: Install Linux Kernel 5.16
+      shell: /usr/local/bin/kernel_update.sh -i 5.16.20
+      args:
+        executable: /bin/bash
+    
+    - name: Update GRUB configuration
+      command: update-grub
+
+    - name: Reboot the system to apply the new kernel
+      reboot:
+        reboot_timeout: 300
+    
+    - name: Wait for the system to come back online
+      wait_for_connection:
+        timeout: 300
+
+    - name: Start the vulnerable service
+      shell: |
+        tmux new-session -d -s vulnerable_service 'cd /usr/local/bin && ./fuse /usr/local/bin/ovlcap/lower /usr/local/bin/gc'
+      args:
+        executable: /bin/bash
+
+    - name: Output instructions for CVEX
```

Comment:
Remove this command. The intention is to have a clean Ansible playbook that could be shared.